Makefile

```diff
@@ -1,5 +1,5 @@
 AUTOTAGS := $(shell ./btrfs_tag.sh) $(shell ./libdm_tag.sh) $(shell ./ostree_tag.sh) $(shell ./selinux_tag.sh)
-TAGS := seccomp
+TAGS ?= seccomp
```

Comment:
Makefile ignorance questtion.  What does the '?=' do vs ':='  ?

Reply:
I had to look this one up in the 'make' info pages: this defines TAGS if it isn't already defined.  We only define TAGS here, so I don't know that we gain anything by it, but it shouldn't break anything, either.

Reply:
Thanks Nalin, I search for 15 or 20 minutes last night and never found it.